include/bitcoin/client/obelisk/obelisk_types.hpp

```diff
@@ -48,8 +40,10 @@ typedef std::vector<history_row> history_list;
 
 struct BCC_API stealth_row
 {
-    data_chunk ephemkey;
-    payment_address address;
+    // No sign byte in public key.
+    hash_digest ephemkey;
+    // No version byte in address.
+    short_hash address;
     hash_digest transaction_hash;
```

Comment:
OK, so we could save it when extracting the hash but that would result in an extra byte per record, and we could recalculate it when returning the query result, but that's a bunch of work. I get it, no prob.